### ISSUE TITLE
Creating a jdoc for this results of this project

### ISSUE THREAD

**Issue by astridx** [joomla-projects/gsoc16_browser-automated-tests#129]
Up to know we have no jdoc that explains testing joomla with codeception. At least I did not find one.

We have a jdoc that explains testing an extension with codeception: https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception.

Should we create a new jdoc for the results of this project or should we integrate this informations into the existing jdoc https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception ?

**Comment by puneet0191** [comment]
I agree with the proposal, I think we can have this as one of the tasks for the Gherkin Editor GSoC Project, Most of the information is clearly available as part of read.me on the repo. What we should do is just have a Wiki created on JDocs for Automated testing and start adding all the links to relevant documentation.

**Comment by puneet0191** [comment]
We have one, this is completely outdated. No more Selenium RC tests, No more Selenium Webdriver Tests https://docs.joomla.org/Running_Automated_Tests_for_the_Joomla_CMS
